src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__stars {
+    background-image: url(../images/star.svg);
```

Comment:
The background-image URL uses a relative path ('../images/star.svg'). Ensure that this path is correct relative to the location of the compiled CSS file, or consider using an absolute path if the image is not displaying correctly.